src/components/arrow.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+
+const spin = keyframes`
+  from { transform: rotate(0deg); }
+  to { transform: rotate(180deg); }
+`;
+
+const spinReverse = keyframes`
+  from { transform: rotate(180deg); }
+  to { transform: rotate(0deg); }
+`;
+
+const ArrowContainer = styled.div`
+	display: inline-block;
+	margin-left: 5px;
+	svg {
+		position: relative;
+		top: -2px;
+		animation: ${props => (props.open ? spinReverse : spin)} 0.3s linear forwards;
+	}
+`;
+
+const Arrow = ({ open }) => (
+	<ArrowContainer open={open}>
+		<svg width="10" height="6" viewBox="0 0 10 6" fill="none" xmlns="http://www.w3.org/2000/svg">
+			<path
```

Comment:
put this in an `arrow.svg` file?

Reply:
Yes! I forgot we actually have an asset for that!